docs/index.md

```diff
@@ -104,7 +104,7 @@ the circuit 2-qubit gate count is not reduced after compilation.
 * - [AutoRebase [2]](inv:#*.AutoRebase)
   - [SynthesiseTket](inv:#*.SynthesiseTket)
   - [FullPeepholeOptimise [3]](inv:#*.passes.FullPeepholeOptimise)
-  - [RemoveBarriers](inv:#*.passes.RemoveBarriers)
```

Comment:
Not sure why this first reference didn't resolve. Tested that this replacement works locally though.

Reply:
I don't know why it wouldn't work. Does this now show the name `_tket` in the docs? I think we want to avoid that.

Reply:
Yeah unfortunately I'll need to fix that in the TKET API docs. There's a replacement of all of the `_tket` names that isn't working currently. However I would've thought this resolver used module imports and that the module names on the webpage wouldn't matter.